src/utils.js

```diff
@@ -301,6 +301,46 @@ export const truncate = (str, opts = {}) => {
   return str;
 };
 
+/*
+Rounding Types:
+  ROUND_UP: 0 - Rounds away from zero
+  ROUND_DOWN: 1 - Rounds towards zero
+  ROUND_CEIL: 2 - Rounds towards Infinity
+  ROUND_FLOOR: 3 - Rounds towards -Infinity
+  ROUND_HALF_UP: 4 - Rounds towards nearest neighbour.
+  If equidistant, rounds away from zero
+  ROUND_HALF_DOWN: 5 - Rounds towards nearest neighbour.
+  If equidistant, rounds towards zero
+  ROUND_HALF_EVEN: 6 - Rounds towards nearest neighbour.
+  If equidistant, rounds towards even neighbour
+  ROUND_HALF_CEIL: 7 - Rounds towards nearest neighbour.
+  If equidistant, rounds towards Infinity
+  ROUND_HALF_FLOOR: 8 - Rounds towards nearest neighbour.
+  If equidistant, rounds towards -Infinity
+*/
+
+export const round = (value, type, decimalPlaces) => {
```

Comment:
Sorry for not being clear in discord. Let me try to be a bit more specific here.

This enum you created here is fine, but BigNumber.js obviously already has the same mapping.  What I was suggesting is that when calling this function (for the app for example),

Instead of 

`$: rewardsBalance = round($global.rewardsBalance, 1, 18)`


we use 
`$: rewardsBalance = round($global.rewardsBalance, BigNumber.ROUND_DOWN, 18);`

so that some reading the code understands what "1" is. 

Let me know if this makes sense.

Reply:
@0xean thanks for the review, I think now I got it